### ISSUE TITLE
How to make this project?

### ISSUE THREAD

**Issue by shriharan18** [liouma/Modifications-on-Ewon-project#1]
Please help me out. I wanna make this project. I am from India and I am 15 years old. I have a basic knowledge in Python and I am new to Raspberry Pi. I contacted SharathNaik to make EWON project as the snowboy is not supported anymore. You have modified the code. Now I need guide to make this project. Please help me out.

**Comment by liouma** [comment]
Sure, I will help you!
What hardware did you use for the display? they use different software.
Liouma

**Comment by shriharan18** [comment]
I didn't start making this project. I am gonna start it. Please connect on
Discord. I have already sent messages to you on Discord. I am Shriharan
from India

**Comment by liouma** [comment]
OK great, we are almost neighbours, I live in a small french island in the
indian ocean :D
You can start following my instructable page :
https://www.instructables.com/EWON-Raspberry-Pi-Powered-Home-Robot-new-Hardware-/
If you want I will guide you step by step and we will finish together
writing the instructions on this page while you advance in making the robot

Liouma Tokitsu

**Comment by shriharan18** [comment]
I am 15 years old doing my high school. I made my first robot at my 7th
grade. I have a huge interest in robotics and so I wish to to do this
project. Now please help me making this project. First what should I start
with. You have used Raspberry Pi Zero W. Is that enough. What is the RAM
capacity of the board?

**Comment by liouma** [comment]
The list of hardware on my instructable is what I chose and it work. You
are free to make other choices
Liouma

**Comment by shriharan18** [comment]
Can I use Raspberry Pi 4?

**Comment by liouma** [comment]
Yes but it will be difficult to fit it inside the body


Liouma

- envoyé depuis un mobile -

**Comment by shriharan18** [comment]
Then, shall I use the 3D print model that Sharath Naik made?

**Comment by liouma** [comment]
No, it will be the same. You will have more space in my model as I includ a
board in the head while Sharu puts everything in the body. So you will have
more free space in the body for the pi.
You can try with your pi, buy my screen on aliexpress, print the parts and
it will work.
The only difference is that your pi is a bit larger than mine but I don't
think there will be other changes. Yours will be more powerful but the pi
zero is enough.
Anyway you can start with our pi and it will be easy to change later if you
want to miniaturize.
Liouma
